Wrapper/DBEngine.cpp

```diff
@@ -0,0 +1,273 @@
+﻿/*
+ * Copyright 2020 OmniSci, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "DBEngine.h"
+#include <thrift/Thrift.h>
+#include <array>
+#include <boost/filesystem.hpp>
+#include <boost/variant.hpp>
+#include <exception>
+#include <iostream>
```

Comment:
Do we really need all that headers here? Including iostream?

Reply:
Yes. 
iostream is used temporary.

Reply:
What about all other libraries? You may use `include-what-you-use` utility to remove unused includes.

Reply:
fixed